{
  GridException me=prepareMultiException();
  try {
    throw new RuntimeException("Test.",me);
  }
 catch (  RuntimeException e) {
    assertFalse(X.hasCause(e,IOException.class));
    assertTrue(X.hasCause(e,GridException.class));
    assertTrue(X.hasCause(e,IllegalArgumentException.class));
  }
}
